demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -336,12 +336,15 @@ def format_converted_pack(self) -> None:
         )
         from_version = "6.0.0" if self.create_new else ""
         format_manager(
+            input=str(self.pack_dir_path),
```

Comment:
These changes were made because if not, format would detect all changes in repo and not only the ones in the modified pack as seen in https://github.com/demisto/contribution-management/actions/runs/7948720170/job/21699063419#step:12:272